packages/boxel/addon/components/boxel/input/selectable-token-amount/index.css

```diff
@@ -0,0 +1,64 @@
+.boxel-input-selectable-token-amount {
+  --boxel-input-selectable-token-amount-height: var(--boxel-sp-xl);
+  --boxel-input-selectable-token-amount-horizontal-padding: var(--boxel-sp-sm);
+  --boxel-input-selectable-token-amount-input-font-size: var(--boxel-font-size);
+
+  position: relative;
+  width: 100%;
+  font-family: var(--boxel-font-family);
+  font-size: var(--boxel-input-selectable-token-amount-input-font-size);
+}
+
+.boxel-input-selectable-token-amount .boxel-input-selectable-token-amount__input {
+  --boxel-input-height: var(--boxel-input-selectable-token-amount-height);
+
+  height: var(--boxel-input-selectable-token-amount-height);
+  padding-left: var(--boxel-input-selectable-token-amount-horizontal-padding);
+  font: inherit;
+  letter-spacing: inherit;
+}
+
+.boxel-input-selectable-token-amount .ember-power-select-status-icon {
+  background: url("/@cardstack/boxel/images/icons/caret-down.svg") no-repeat;
+  width: 11px;
+  height: 9px;
+  display: inline-block;
+}
+
+.boxel-input-selectable-token-amount .ember-basic-dropdown {
+  display: contents;
+}
+
+.boxel-input-selectable-token-amount__select {
+  position: absolute;
+  height: var(--boxel-input-selectable-token-amount-height);
+  display: flex;
+  align-items: center;
+  top: 0;
+  right: 0;
+  user-select: none;
+  padding-right: var(--boxel-input-selectable-token-amount-horizontal-padding);
+}
+
+.boxel-input-selectable-token-amount__select .boxel-input-selectable-token-amount__dropdown-item.boxel-select__item,
+.boxel-input-selectable-token-amount__dropdown .boxel-input-selectable-token-amount__dropdown-item.boxel-select__item {
+  display: flex;
+  align-items: center;
+  column-gap: var(--boxel-sp-xxs);
+  padding: var(--boxel-sp-xs) var(--boxel-sp-sm);
+  font-size: var(--boxel-font-size-sm);
+  font-weight: 600;
+}
+
+.boxel-input-selectable-token-amount__icon {
+  width: var(--boxel-icon-sm);
+  height: var(--boxel-icon-sm);
+}
+
+.boxel-input-selectable-token-amount__select[aria-disabled="true"] {
+  opacity: 0.5;
+}
+
+.boxel-input-selectable-token-amount__select[aria-disabled="true"] .ember-power-select-selected-item {
```

Comment:
The cursor on the caret is still "pointer" when disabled. I think there's a Boxel rule that is targeting disabled buttons this way:

```css
button:hover:not(:disabled),
[role="button"]:hover:not(:disabled) {
  cursor: pointer;
}
```

We could change it to also include `[aria-disabled="true"]`?

```css
button:hover:not(:disabled), 
[role="button"]:hover:not(:disabled, [aria-disabled="true"]) {
  cursor: pointer;
}
```

In which case we wouldn't need this rule, I think.

Reply:
much better, thank you!